src/presenter/movieListPresenter.js

```diff
@@ -5,7 +5,7 @@ import FilterView from '../view/filter';
 import ShowMoreButtonView from '../view/showMoreBtn';
 import FilmPresenter from './filmPresenter';
 
-import {render, isEmptyData, remove, updateItem} from '../utils.js';
+import {render, isEmptyData, remove, updateItem, SortType, getDateDiff, getRatingDiff} from '../utils.js';
```

Comment:
Не надо сокращать слова, это нарушение критерия

```suggestion
import {render, isEmptyData, remove, updateItem, SortType, getDateDifference, getRatingDifference} from '../utils.js';
```